packages/primitives/package.json

```diff
@@ -2,10 +2,9 @@
   "name": "@strapi/ui-primitives",
   "version": "1.9.0",
   "license": "MIT",
-  "type": "module",
```

Comment:
Why did you have to remove `type: module`? 🤔

Reply:
This is a bit interesting, so following work on `react-spring` around the package.json etc. There was surprisingly a twitter thread at the same time i was _lightly_ involved in with some interesting people – https://twitter.com/devongovett/status/1629161014812897282?s=20 anyway, an output of that was that at this point, using `type: module` would cause more headaches than it's worth when you're trying to support CJS & ESM at the same time, something imo we should be doing because the ecosystem is still very fractured.

You can read the redux maintainer's blog post on the subject [here](https://blog.isquaredsoftware.com/2023/08/esm-modernization-lessons/) which was equally as interesting.

the TLDR; is that adding `type: module` was more of a headache for a variety of bundlers which whilst this isn't a package 1million people use in 100000 environments I'd rather support all the things properly so we're not having issues if we make a jump to X or the project evolves etc. 👍🏼